docs/management/updates.md

```diff
@@ -113,6 +113,23 @@ Also known as RPU, **this is the advised way to update your pool**. By just clic
 This powerful and fully automated mechanism requires some prerequisites: all your VMs disks must be on a one (or more) shared storage. Also, high-availability will be automatically disabled, as the XO load balancer plugin and backup jobs. Everything will be enabled back when it's done!
 :::
 
+:::warning
+RPU is disabled for pools with LINSTOR SRs. The reason is that after a reboot of a just updated host, it's possible that it can no longer communicate with other hosts through LINSTOR satellites. In fact LINSTOR expects that we always use satellites and controllers with the same version.
+
+To avoid problems, it is strongly recommended to update the satellites, controllers packages of each host without rebooting:
+```
+yum update linstor-satellite linstor-controller
+```
+
+After updating all hosts without reboot:
+```
+yum restart linstor-satellites
+yum stop linstor-controller # No error here, use "stop".
+```
+
+Then you can follow the instructions in the documentation to manually update the pool.
```

Comment:
In theory, could we use RPU after doing the linstor package updates? If yes, that's a potential solution to have RPU improved to always try to update linstor first on all hosts, and when done, doing the regular RPU algorithm.

Reply:
Yes. But anyway we want to automate this, this document is supposed to be temporary. We need to plan a meeting with Mathieu and Samuel probably to confirm the algorithm and see if we don't have an edge case: Is it wiser to update linstor RPMs before or after RPU for example.